core/controllers/blog_admin.py

```diff
@@ -114,11 +114,15 @@ def post(self):
                 'action') == 'save_config_properties':
             new_config_property_values = self.normalized_payload.get(
                 'new_config_property_values')
+            if new_config_property_values[
+                    'max_number_of_tags_assigned_to_blog_post'] == 0:
```

Comment:
What if `max_number_of_tags_assigned_to_blog_post` < 0?

Can you please add a schema for `max_number_of_tags_assigned_to_blog_post` which will  not allow add invalid values?

Reply:
Changed to <= 0 and added test case for the same. All other schema validations are already performed  by the schema provided by the config domain in the controller args schema. Thank you.